File: Easy/Python/2656. Maximum Sum With Exactly K Elements.py
class Solution(object):
    def maximizeSum(self, nums, k):
        count = 0
        maximum = 0

        for i in range(len(nums)):
            maximum = max(maximum, nums[i])
        
        while 0 < k:
            count += maximum
            maximum += 1
            k -= 1

        return count

File: Easy/Java/2656. Maximum Sum With Exactly K Elements.java
class Solution {
    public int maximizeSum(int[] nums, int k) {
        int count = 0;
        int max = 0;

        for(int i = 0; i < nums.length; i++){
            max = Math.max(max, nums[i]);
        }
        
        while(0 < k){
            count += max;
            max++;
            k--;
        }

        return count;
    }
}
